src/ray/gcs/accessor.h

```diff
@@ -589,7 +589,6 @@ class NodeResourceInfoAccessor {
  protected:
   NodeResourceInfoAccessor() = default;
 
- private:
```

Comment:
We’d better to keep this `private` flag. There's a method called `GetLastResourceUsage` for accessing this var.

Reply:
I made the following changes: `GetLastResourceUsage` returns the const pointer to prevent external modification of the value of `last_resource_usage_`, thx.